internal/schema.go

```diff
@@ -141,6 +141,23 @@ type IDPEntityDescriptor struct {
 	ASSERTION        string           `xml:"xmlns:assertion,attr"`
 	EntityId         string           `xml:"entityID,attr"`
 	IDPSSODescriptor IDPSSODescriptor `xml:"IDPSSODescriptor"`
+	Organization *Organization `xml:"Organization,omitempty "`
+	ContactPerson []ContactPerson `xml:"ContactPerson,omitempty "`
+}
+
+type Organization struct {
+	XMLName xml.Name
+	OrganizationName     string `xml:"xmlns:organizationname,attr"`
+	OrganizationDisplayName  string `xml:"xmlns:organizationdisplayname,attr"`
+	OrganizationURL   string `xml:"xmlns:organizationurl,attr"`
```

Comment:
@sccalabr 
1. these 3 properties are not attributes but `attr` used in the `xml:` 
2. Also these property having no namespace, but you have used `xmlns` namespace

same issues with Below Contactperson struct